wezterm-gui/src/overlay/quickselect.rs

```diff
@@ -616,6 +623,8 @@ impl Pane for QuickSelectOverlay {
                             }
                         }
                         line.clear_appdata();
+                    } else if disable_attr {
+                        line.clear_appdata();
```

Comment:
should this go together with the attribute cleaing on line 559? Feels like it should, but not sure if you put it here because it only needs to go here